src/components/BuilderRoute.tsx

```diff
@@ -9,26 +9,30 @@ import { useCurrentNodeContext } from './CurrentNodeProvider';
 const BuilderRoute: FC = () => {
   const { id, nodeId } = useParams();
   const { pathways } = usePathwaysContext();
-  const { setPathway } = useCurrentPathwayContext();
+  const { pathwayRef, setPathway, resetPathway } = useCurrentPathwayContext();
```

Comment:
Looks like it was already this way, but it might be less confusing if `setPathway` was `setCurrentPathway`.  I know some of this will likely change when the backend is implemented so maybe it's not worth it.

What do you think?

Reply:
Yea I was contemplating changing it due to my own laziness didn't - I shall go an update now though!